demo/api/src/products/custom-product.resolver.ts

```diff
@@ -1,18 +1,45 @@
 import { RequiredPermission } from "@comet/cms-api";
 import { InjectRepository } from "@mikro-orm/nestjs";
 import { EntityRepository } from "@mikro-orm/postgresql";
-import { Mutation, Resolver } from "@nestjs/graphql";
+import { Mutation, Query, Resolver } from "@nestjs/graphql";
+import { getMessages } from "@src/common/lang";
+import { createIntl, createIntlCache, IntlCache } from "react-intl";
 
 import { Product, ProductStatus } from "./entities/product.entity";
 
 @Resolver(() => Product)
 @RequiredPermission(["products"], { skipScopeCheck: true })
 export class CustomProductResolver {
-    constructor(@InjectRepository(Product) private readonly repository: EntityRepository<Product>) {}
+    private readonly intlCache: IntlCache;
+    constructor(@InjectRepository(Product) private readonly repository: EntityRepository<Product>) {
+        this.intlCache = createIntlCache();
+    }
 
     @Mutation(() => Boolean)
     async publishAllProducts(): Promise<boolean> {
         await this.repository.nativeUpdate({ status: { $ne: ProductStatus.Published } }, { status: ProductStatus.Published });
         return true;
     }
+
+    @Query(() => String)
+    async myTranslatedString(): Promise<string> {
+        const language = "de"; // TODO correctly set the language, maybe move into the mail template
```

Comment:
How is it planned to get the desired Language? 

I currently see two possibilities:
1. Language is stored in a database
2. Language gets dictated from the API call (Client)
    - Could be an input argument or a HTTP header - `Accept-Language`